app/src/main/java/com/github/sdpteam15/polyevents/database/FakeDatabase.kt

```diff
@@ -2,23 +2,26 @@ package com.github.sdpteam15.polyevents.database
 
 import android.os.Build
 import androidx.annotation.RequiresApi
+import com.github.sdpteam15.polyevents.database.observe.Observable
 import com.github.sdpteam15.polyevents.event.Event
 import com.github.sdpteam15.polyevents.user.ProfileInterface
 import com.github.sdpteam15.polyevents.user.UserInterface
 import java.time.LocalDateTime
+import java.util.*
+import kotlin.collections.ArrayList
 
 object FakeDatabase : DatabaseInterface {
     init {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O)
             initEvents()
     }
 
-    private var events: ArrayList<Event>? = null
+    var events: ArrayList<Event>? = null
```

Comment:
Is it possible to do this with a lateinit, instead of asserting non null with !! at each invocation of events?